--disable-test causes packetmill linker issue

Configuring Fastclick with `./configure --disable-test` followed by `sudo make install` on Debian 14 causes the building process to fail and produces the following output : 

```
  CXXLINK packetmill.cc
/usr/bin/ld: /tmp/ccRkDmPg.o: in function `parse(int, char**, click_args_t&)':
/home/vboxuser/fastclick/userlevel/args.hh:144: undefined reference to `Clp_NewParser'
/usr/bin/ld: /home/vboxuser/fastclick/userlevel/args.hh:145: undefined reference to `Clp_ProgramName'
/usr/bin/ld: /home/vboxuser/fastclick/userlevel/args.hh:148: undefined reference to `Clp_Next'
/usr/bin/ld: /tmp/ccRkDmPg.o: in function `parse(int, char**, click_args_t&)':
/home/vboxuser/fastclick/userlevel/packetmill.cc:21: undefined reference to `Clp_DeleteParser'
/usr/bin/ld: /home/vboxuser/fastclick/userlevel/packetmill.cc:21: undefined reference to `Clp_DeleteParser'
/usr/bin/ld: /home/vboxuser/fastclick/userlevel/packetmill.cc:21: undefined reference to `Clp_DeleteParser'
/usr/bin/ld: /tmp/ccRkDmPg.o: in function `parse(int, char**, click_args_t&)':
/home/vboxuser/fastclick/userlevel/args.hh:203: undefined reference to `Clp_OptionError'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:214: packetmill] Error 1
make[1]: Leaving directory '/home/vboxuser/fastclick/userlevel'
make: *** [Makefile:71: install-userlevel] Error 2
```

This seems to be related to clp being excluded from the building process, despite `clp.o` appearing in the trace

[trace.txt](https://github.com/user-attachments/files/17561185/trace.txt)

--- a/userlevel/Makefile.in
+++ b/userlevel/Makefile.in
@@ -99,7 +99,7 @@ LLVM_OBJS = @LLVM_OBJS@
 LIBOBJS = $(GENERIC_OBJS) $(STD_ELEMENT_OBJS) clp.o exportstub.o
 STD_ELEMENT_OBJS = addressinfo.o alignmentinfo.o \
 	errorelement.o portinfo.o scheduleinfo.o
-OBJS = $(ELEMENT_OBJS) $(ELEMENTSCONF)$(ELEMENTSSUFFIX).o click.o
+OBJS = $(ELEMENT_OBJS) $(ELEMENTSCONF)$(ELEMENTSSUFFIX).o clp.o click.o
 
 CPPFLAGS = @CPPFLAGS@ -DCLICK_USERLEVEL
 CFLAGS = @CFLAGS@

